### DISCUSSION TITLE
Hot to use downloadFrom form field?

### DISCUSSION THREAD

**Discussion by starwalkn** [gotenberg/gotenberg#982] [Q&A] [answered]
Hi! To begin with, are there any examples of using this formField? Now I don't quite understand how the client should work with it, for example, with a regular HTML request to generate a PDF.
Does formField allow you to download the HTML source file from the downloadFrom URL?

**Comment by gulien** [comment, marked as answer]
Hello @dcaraxes 👋 

From a client perspective, it's simply a new form field. Checkout my commit on the PHP client: https://github.com/gotenberg/gotenberg-php/commit/362845ba30147cf31ccdd0c41a82922f957ac7b2 😉 

> Does formField allow you to download the HTML source file from the downloadFrom URL?

Exactly! It's just an alternative (which is not exclusive) of form files. See also https://github.com/gotenberg/gotenberg/issues/894.
